### DISCUSSION TITLE
How to implement a fade-out animation for the GPUI component?

### DISCUSSION THREAD

**Discussion by eastack** [zed-industries/zed#17059] [Feature Requests]
In another project using GPUI, I have the same question: Can GPUI implement a fade-out animation when a component is closed, and if so, how? 

https://github.com/huacnlee/gpui-component/issues/148

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
